src/clauses/Call.ts

```diff
@@ -100,6 +101,14 @@ export class Call extends Clause {
         return this;
     }
 
+    /** Makes the subquery an OPTIONAL CALL
+     * @see [Cypher Documentation](https://neo4j.com/docs/cypher-manual/current/subqueries/call-subquery/#optional-call)
+     */
+    public optional(): this {
+        this._optional = true;
```

Comment:
Just a consideration, even if this is what is already happening with `OptionalMatch`, by doing this, we will have both instances of `OptionalCall` and `Call` that could produce the same optional Cypher. We could change this to be: `new OptionalCall()`.

Reply:
The reason I haven't done it that way, is that then the cypher builder tree would look like this:

CALL
 --- > OptionalCall
 
 I'm sure that there is a way to improve on that that we could discuss, but this works well enough

Reply:
Mmm true, nothing will prevent keeping using the previous `Call` reference before `optional` is called. Another way could be to accept optional in the constructor or have an Optional class that could accept both `MATCH` and `CALL`. Anyway, I agree that works already fine, just a consideration!